packages/date/README.md

```diff
@@ -22,14 +22,13 @@ Formats a date (like `date()` in PHP).
 
 _Related_
 
--   <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>
--   <https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC>
+-   <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones-> <https://en.wikipedia.org/wiki/ISO_8601#Time_offsets_from_UTC>
```

Comment:
Can we split it into multiple lines again?

Reply:
Whoops, yes! I found the bug I introduced that caused this.